simulariumio/physicell/physicell_converter.py

```diff
@@ -38,15 +38,16 @@ def __init__(self, input_data: PhysicellData):
         self._data = self._read(input_data)
 
     @staticmethod
-    def _load_data(path_to_output_dir) -> np.ndarray:
+    def _load_data(path_to_output_dir: str, nth_timestep_to_read: int) -> np.ndarray:
```

Comment:
should there be a default value for the new arg?

Reply:
this defaults to 1 in `PhysicellData` but this function isn't part of the public API and will probably only be called internally to `PhysicellConverter` which always provides it, so I didn't add it